### ISSUE TITLE
hi~I got a problem with it,when I use MBProgressHUD for its customView

### ISSUE THREAD

**Issue by zeku2022** [ninjaprox/NVActivityIndicatorView#62]
when I use NVActivityIndicatorView as the customView,its frame can't changed,just look like this,I want the activityIndicatorView move to the center of the HUD.,What's the problem?Please help me !!thx a lot!
![qq20160906-1](https://cloud.githubusercontent.com/assets/11352346/18265531/c8c64946-7448-11e6-86e8-a34edd0227df.png)

the code:

``` swift
let hud = MBProgressHUD.showHUDAddedTo(view, animated: true)
hud.mode = .CustomView
let activityIndicatorView = NVActivityIndicatorView(frame: CGRectMake(0, 0, 50, 50), type: NVActivityIndicatorType.LineScale)
activityIndicatorView.startAnimation()
hud.customView = activityIndicatorView
hud.square = true
hud.label.text = "test words..."
hud.hideAnimated(true, afterDelay: delayTime)
```

**Comment by ninjaprox** [comment]
@xiaoxiongke: This caused by `intrinsicContentSize`. You can see this note in documentation of `customView` property. `intrinsicContentSize` is used by auto layout system to to calculate constraints and this is exactly what `MBProgressHUD` uses to layout itself and its subviews (`customView` is subview of `bezel` view, the rectangle). `NVActivityIndicatorView` hadn't, however,  returned this value properly, it was (-1, -1) by default, especially when `UIView` is created by code.

Fix is on `master` now and would be included in next version. Thanks for reporting this!

**Comment by ninjaprox** [comment]
This is fixed in version [2.11](https://github.com/ninjaprox/NVActivityIndicatorView/releases/tag/v2.11).
